Branch-SDK/src/main/AndroidManifest.xml

```diff
@@ -2,4 +2,11 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     android:versionCode="1"
     android:versionName="1.0" >
+    <queries>
```

Comment:
This should be left to the integrating app, the SDK cannot be adding this by default.